causalml/propensity.py

```diff
@@ -15,19 +15,22 @@ class ElasticNetPropensityModel(object):
         model (sklearn.linear_model.ElasticNetCV): a propensity model object
     """
 
-    def __init__(self, n_fold=5, clip_bounds=(1e-3, 1 - 1e-3), random_state=None):
+    def __init__(self, n_fold=5, clip_bounds=(1e-3, 1 - 1e-3), l1_ratios=np.linspace(0,1,5), random_state=None):
```

Comment:
Let's allow users to pass `cv` as an input argument. We can set it to `None` by default and if not given, we can generate it as in line 31.

Reply:
Done!